setup.cfg

```diff
@@ -1,5 +1,13 @@
 [flake8]
 max-line-length = 120
-exclude = docs,env,*.egg,migrations
+exclude = docs,env,*.egg,migrations,south_migrations
```

Comment:
sorry to throw everything back and forth, but since we're not supporting 1.6, we won't need `south_migrations`